paket.dependencies

```diff
@@ -49,7 +49,7 @@ nuget Expecto.Diff
 nuget YoloDev.Expecto.TestSdk
 nuget AltCover
 nuget GitHubActionsTestLogger
-nuget Ionide.LanguageServerProtocol
+nuget Ionide.LanguageServerProtocol 0.4.8
```

Comment:
this'll need to be bumped (or the explicit versions removed from this and the ProjInfo deps) prior to merge I think